Index: appdriver/src/main/java/org/rmj/g3appdriver/GConnect/Marketplace/Payment/MpPayment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/appdriver/src/main/java/org/rmj/g3appdriver/GConnect/Marketplace/Payment/MpPayment.java b/appdriver/src/main/java/org/rmj/g3appdriver/GConnect/Marketplace/Payment/MpPayment.java
--- a/appdriver/src/main/java/org/rmj/g3appdriver/GConnect/Marketplace/Payment/MpPayment.java	
+++ b/appdriver/src/main/java/org/rmj/g3appdriver/GConnect/Marketplace/Payment/MpPayment.java	
@@ -11,5 +11,7 @@
         this.instance = instance;
     }
 
-    public boolean PayOrder(String TransNo)
+    public boolean PayOrder(String TransNo) {
+        return false;
+    }
 }
